Javascript/statistic.js

```diff
@@ -30,10 +30,10 @@ function loadStatisticsMultiplier() {
 }
 
 function loadStatisticsCubeMultipliers() {
-    let arr = calculateCubeMultiplier(false)
-    let acc = [2, 4, 4, 1, 1, 1, 1, 1, 2, 3, 3, 3, 3, 4, 4, 5, 4]
+    const arr = calculateCubeMultiplier(false).slice(0, -1);
```

Comment:
A proper fix to this would be to add the missing element into index.html instead of chopping off the last element of the array. I have implemented this fix in #138

Reply:
isn't meant to be a proper fix, just a fix in general